tests/validation/tests/v3_api/test_logging_e2e.py

```diff
@@ -0,0 +1,224 @@
+                              "persistence.enabled": "false",
+                              "extraPersistence.enabled": "false",
+                              "extraPersistence.size": "10Gi",
+                              "extraPersistence.mountPath": "/extra",
+                              "extraPersistence.storageClass": "",
+                              "output.type": "custom",
+                              "output.flushInterval": "5s",
+                              "output.customConf": "<match **.**>\n  @type stdout\n</match>"}
+
+FLUENTD_AGGREGATOR_CATALOG_ID = "catalog://?catalog=library&template=fluentd-aggregator&version=0.3.1"
+
+
+fluentd_app_name = "rancher-logging"
+endpoint_port = "24224"
+weight = 100
+
+
+def test_fluentd_target_logs(setup_fluentd_aggregator, request):
+    cluster_logging = create_cluster_logging(fluentd_target_without_ssl())
+    request.addfinalizer(lambda: delete_cluster_logging(cluster_logging))
+    wait_for_logging_app()
+
+    # wait for config to sync
+    time.sleep(90)
+    validate_websocket_view_logs()
+
+
+def test_project_fluentd_target_logs(setup_fluentd_aggregator, request):
+    project_logging = create_project_logging(fluentd_target_without_ssl())
+    request.addfinalizer(lambda: delete_project_logging(project_logging))
+    wait_for_logging_app()
+
+    # wait for config to sync
+    # wait for project logs to start being forwarded
+    time.sleep(90)
+    validate_websocket_view_logs()
+
+
+def wait_for_logging_app():
+    sys_p_client = namespace["sys_p_client"]
+    wait_for_app_to_active(sys_p_client, fluentd_app_name)
+
+
+def fluentd_target_without_ssl():
+    return {"compress": True,
+            "enableTls": False,
+            "sslVerify": False,
+            "fluentServers": [
+                {
+                    "endpoint": namespace["hostname"]+ ":" + endpoint_port,
+                    "weight": weight
+                }
+            ],
+            }
+
+
+def get_system_project_client():
+    cluster = namespace["cluster"]
+    admin_client = namespace["admin_client"]
+    projects = admin_client.list_project(name="System",
+                                         clusterId=cluster.id).data
+    assert len(projects) == 1
+    project = projects[0]
+    sys_p_client = get_project_client_for_token(project, USER_TOKEN)
+    return sys_p_client
+
+
+def create_cluster_logging(config, json_parsing=False):
+    cluster = namespace["cluster"]
+    admin_client = namespace["admin_client"]
+    name = random_test_name("fluentd")
+    return admin_client.create_cluster_logging(name=name,
+                                               clusterId=cluster.id,
+                                               fluentForwarderConfig=config,
+                                               enableJSONParsing=json_parsing,
+                                               outputFlushInterval=5
+                                               )
+
+
+def delete_cluster_logging(cluster_logging):
```

Comment:
This method and `delete_project_logging` are the same -- rename to `delete_logging` and remove one.